internal/sum/sum.go

```diff
@@ -0,0 +1,279 @@
+package sum
```

Comment:
is `sum` an acronym?

Reply:
Yes, Supermicro Update Manager https://www.supermicro.com/en/solutions/management-software/supermicro-update-manager

Reply:
thanks, please make a package level code comment so we dont lose this info once the PR is merged.